projects/openjdk.org/package.yml

```diff
@@ -144,6 +152,8 @@ provides:
   - bin/jstack
   - bin/jstat
   - bin/jstatd
+  # TODO: if: {{version.major}} >= 21
+  # - bin/jwebserver
```

Comment:
Any ideas about this?

Reply:
just need to omit it; it would be a potential upgrade to libpkgx if desired.

Reply:
https://github.com/pkgxdev/libpkgx/issues/68